code/__HELPERS/game.dm

```diff
@@ -236,6 +236,9 @@
 	for(var/mob/candidate_mob as anything in group)
 		if(!candidate_mob.key || !candidate_mob.client || (ignore_category && GLOB.poll_ignore[ignore_category] && (candidate_mob.ckey in GLOB.poll_ignore[ignore_category])))
 			continue
+		for(var/content_preference in subtypesof(/datum/preference/choiced/content))
+			if(candidate_mob.client.prefs.read_preference(content_preference) == "Unset")
```

Comment:
I feel as if every instance of checking for a string that's a yes or no should be a define for that strings value.